### DISCUSSION TITLE
[SUGGESTION] Modul Npc-Bot (Botcore)

### DISCUSSION THREAD

**Discussion by Selyria** [azerothcore/azerothcore-wotlk#3934] [Ideas]
Heyho I have a suggestion how would it be if you could make a module with which you could create bot Npc. 

A video how this could look like

https://www.youtube.com/watch?v=aZJQk-U2Wng

http://www.ac-web.org/forums/showthread.php?188884-BotCore-335-amp-434

**Comment by BarbzYHOOL** [comment]
Some attempts have been made on the singlecore repack from Conan

**Comment by Psilovybin** [comment]
I've got bots working on an older core but i can't get them working in the newer core

**Comment by BarbzYHOOL** [comment]
share them so someone can fix them maybe?

**Comment by ELdoBA** [comment]
> I've got bots working on an older core but i can't get them working in the newer core

Could you share it please?

**Comment by Psilovybin** [comment]
> share them so someone can fix them maybe?



> > I've got bots working on an older core but i can't get them working in the newer core
> 
> Could you share it please?

https://github.com/Psilovybin/azcore-bots It's finished uploading.. tell me what you think. I got frustrated and just left for a while. I've tried to take the updated azcore and merge them with what's in this version, but there's too many internal code changes and worldengine and drassil is already integrated into the new core so i was busy trying to hash those out.  I may be going about it the wrong way but I don't put the pro in programmer if you know what I mean. :)  Just enough to get me into trouble

**Comment by BarbzYHOOL** [comment]
Hmm interesting, this should be shared in discord.


How good did they work on that core ?

**Comment by BarbzYHOOL** [comment]
https://github.com/azerothcore/forum/issues/52
